api/nakadi-event-bus-api.yaml

```diff
@@ -1270,3 +1281,43 @@ definitions:
         description: Documentation for the enrichment.
     required:
       - name
+
+
+  BatchItemResponse:
+    description: |
+      A status corresponding to one individual Event's publishing attempt.
+    properties:
+      eid:
+        type: string
+        format: uuid
+        description: |
+          eid of the corresponding item.
+      publishing_status:
+        type: string
+        enum:
+          - FAILED
+          - SUBMITTED
+          - ABORTED
+        description: |
+          Indicator of the submission of the Event. SUBMITTED indicates successful submission, including commit on he underlying broker. FAILED indicates the message submission was not possible and can be resubmitted if so desired.
+
+          ABORTED indicates that no further attempt at publishing the affected item was attempted because of a failure on another item.
```

Comment:
Do we need to include information about the source of the problem in case of the `ABORTED`?